src/windshaft/map-serializer/anonymous-map-serializer/layers-serializer.js

```diff
@@ -77,6 +77,14 @@ function optionsForMapnikLayer (layerModel) {
     };
   }
 
+  if (isFinite(layerModel.get('minzoom'))) {
```

Comment:
Since the default value is `0` we are always sending `minzoom`. I don't know if it has implications in server side but just in case, I'd do that same thing you've done with `maxzoom`, leaving it `undefined`.

Reply:
According to [this](https://github.com/CartoDB/Windshaft/blob/master/doc/MapConfig-1.7.0.md#2-file-format) `undefined` means infinite and is the default value for maxZoom.

I just copied the default values from windshaft